custom_theme/boxes.html

```diff
@@ -52,10 +52,16 @@ <h2>The easiest way to get started</h2>
       <div class="col-xs-12 col-md-6 col-lg-4">
           <a href="{{ box.displayName }}/">
             <div class="card">
+<<<<<<< HEAD
```

Comment:
hey @leeftk looks like there's a bit of an issue with the merging here (and below)

Reply:
Rebased and fixed these!